code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -103,7 +103,7 @@ \subsection{Table of Symbols}
 \\
 $L$ & Length & m
 \\
-$M$ & Total Mass of the Rigid Body & kg
+$M$ & Mass of the Largest Rigid Body & kg
```

Comment:
I think this would be better if we said the mass of the "larger" rigid body, since you are only comparing two masses when this comes up.

Reply:
yeah, makes sense. I will update